### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniel-Kigozi**
#### In this capstone, we implemented the following as per the project requirements:

The home page.
- When the page loads, the web app retrieves data from:
  - [TVmaze API](https://www.tvmaze.com/api): to show data about TV series and movies.
  - The Involvement API to deliver the item likes.
- The page makes 2 requests:
  - One to the base API.
  - One to the Involvement API.
- When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.

The comments popup.
- When the user clicks on the Comments button, the comments popup appears and the web app retrieves data from:
   - The base API that shows details about the selected item.
   - The Involvement API to deliver the item comments.
- When the user clicks on the Comment button, the data is recorded in the Involvement API and the screen is updated.

To make website dynamic and build basic single page app:
 - Used ES6 syntax, and modules.
 - Used callbacks and promises.
- Used webpack.
- Used AAA pattern for unit tests with Jest.

The home page header/footer and navigation are similar to the given mockup.
Resolved simple git conflicts.
Perform code reviews for team members pull requests.
Updated Readme file with video presentation link.

#### General requirements checklist.

- [X] No linters error.
- [X] Used the correct Gitflow workflow.
- [X] documented work professionally.
- [X] Followed the best practices of HTML, CSS, and JavaScript.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		<nav class="navbar">
+			<div class="logo-container">
+				<a href="#" class="logo">Netflix</a>
+			</div>
+			<ul class="primary-navigation">
+				<li class="navigation-tab"><a href="#">Home</a></li>
+				<li class="navigation-tab"><a href="#">TV Shows</a></li>
+				<li class="navigation-tab"><a href="#">Movies</a></li>
+			</ul>
+		</nav>
```
Comment (KDlamini) [inline src/index.html:23]: On the navigation, kindly create a counter that shows the total number shows that are displayed on the homepage as it is on the [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md).

image

Your navigation:

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @Daniel-Kigozi, @MNisarAli 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a web app with API integration. 🎊 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Project should use ES6 and Webpack.
✔️  The web app uses an API of choice and the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions
✔️  Interface contains a home page, likes, and comments popup.
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
 - [ ] Kindly write a more detailed Pull Request description. Surely there are plenty of features that were implemented in this project that can be listed in bullet points. You can use the kanban tasks or project requirements as features that were implemented.
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,139 @@
...
\ No newline at end of file
+# JavaScript-group-capstone
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```
[the remark this answers is not in the document]
Comment (MNisarAli) [reply README.md]: Removed the Microverse logo.

File: src/index.html (hunk 1 above)
Comment (MNisarAli) [reply src/index.html]: Fixed tv shows counter UI display issues and menu items gap.

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @Daniel-Kigozi & @MNisarAli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**